packages/unicourse/src/unicourse.ts

```diff
@@ -195,13 +195,18 @@ export class UniCourse {
         return this.req(`profile/${username}`);
     }
 
+    /**
+     * @param json The `CoursePack` to import.
+     * @param scope If specified, the importer will use scoped merge strategy with the given id.
+     */
     public async import(
-        json: CoursePack
+        json: CoursePack,
+        scope?: string
```

Comment:
Maybe we could declare a type `ImportOption` as second param?
So that additional arguments won't break function signature and keep `import` simpler. (Just like `fetch` API)

Reply:
I don't think the function signature was broken by this change since `scope` is an optional parameter and defaults to `undefined`.

But I agree that we may need an `ImportOption` in the future once we support more merge strategies. I prefer a new issue for that since it is not a high priority task.